.github/workflows/go.yml

```diff
@@ -21,12 +22,12 @@ jobs:
     steps:
     - name: Check out code
       id: checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@722adc63f1aa60a57ec37892e133b1d319cae598 # Pinned at v2.0.0
```

Comment:
Seems like the perpetual tradeoff between making it easy to get updates, vs pinning to something known.

I haven't seen much discussion about the risk here, and to be honest, was assuming that the `actions` org was somewhat officially blessed by github. 

I kinda wish there was a signed tag pattern or something

Reply:
I think the actions org is Github employees, but that doesn't prevent someone going rogue 🤷